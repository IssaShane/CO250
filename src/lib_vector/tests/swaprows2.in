swaprows [0 1 2; 0 3 6; 0 6 12] 1 2
quit
