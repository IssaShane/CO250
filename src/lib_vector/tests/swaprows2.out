0 1 2
0 6 12
0 3 6
